packages/create-playwright/assets/.eslintrc.js

```diff
@@ -0,0 +1,6 @@
+module.exports = {
```

Comment:
drive-by: migrated instead of ignoring the assets I only disable the notice rule.